lib/wibox/container/background.lua

```diff
@@ -22,7 +22,7 @@ local background = { mt = {} }
 
 -- Draw this widget
 function background:draw(context, cr, width, height)
-    if not self._private.widget or not self._private.widget.visible then
+    if not self._private.widget or not self._private.widget._private.visible then
```

Comment:
Shouldn't this use `:get_visible()` instead? Looking into something called `_private` on another widget seems wrong. (Same comment for other places)